packages/sit-onyx/src/components/OnyxNavButton/OnyxNavButton.stories.ts

```diff
@@ -0,0 +1,84 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import { h } from "vue";
+import OnyxNavButton from "./OnyxNavButton.vue";
+import OnyxListItem from "../OnyxListItem/OnyxListItem.vue";
+import OnyxBadge from "../OnyxBadge/OnyxBadge.vue";
+
+/**
+ * The nav button is used internally to build the main navigation bar component and is not intended to be used individually.
+ */
+const meta: Meta<typeof OnyxNavButton> = {
+  title: "future/NavButton",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxNavButton,
+    events: ["click"],
+    argTypes: {
+      default: {
+        control: { disable: true },
```

Comment:
```suggestion
        control: { type: "text" },
```

Like this, the user can change the slot content as text.
But you also need to remove the `() =>` in the stories below for the default slot for it to work correctly